grpc/sa-wrappers.go

```diff
@@ -283,6 +283,20 @@ func (sac StorageAuthorityClientWrapper) PreviousCertificateExists(
 	return exists, err
 }
 
+func (sac StorageAuthorityClientWrapper) AddPrecertificate(
+	ctx context.Context,
+	req *sapb.AddCertificateRequest,
+) (*corepb.Empty, error) {
+	return sac.inner.AddPrecertificate(ctx, req)
```

Comment:
I think this wrapper needs to check that `req` is `!= nil` and return an error if it is. The `sac.inner.AddPrecertificate` implementation is de-referencing `req` without checking that.

Reply:
Done (and below) :green_apple: